avg_counter.go

```diff
@@ -44,7 +49,7 @@ func (s AvgCounterMetric) GetAvg(name string) (int64, error) {
 
 	for i := 0; i < s.interval; i += 1 {
 		key := getKeyByDate(
-			now.Add(-1 * time.Minute * time.Duration(i)), "avg_counter", name)
+			now.Add(-1*time.Minute*time.Duration(i)), s.globalPrefix+"avg_counter", name)
```

Comment:
Maybe add globalPrefix to `getKeyByDate`'s and `getKey` args?

Reply:
Second argument is already "prefix", two prefix arguments are too complicated, in my opinion.

Reply:
And we can have different separators, and so we can set it right away when calling NewMetrics and then just concatenate with another prefix